src/scripts/domain/moveCalculator.ts

```
@@ -582,6 +582,8 @@ export class MoveCalculator {
       this.addPawnAttackMoves(board, moves, piece);
     }
 
+    console.log('Found moves: ' + moves.length);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)